continuum/scenarios/segmentation.py

```diff
@@ -0,0 +1,220 @@
+import warnings
+from copy import copy
+from typing import Callable, List, Union, Optional
+import os
+import multiprocessing
+
+import numpy as np
+from PIL import Image
+import torchvision
+
+from continuum.datasets import _ContinuumDataset
+from continuum.scenarios import ClassIncremental
+from continuum.tasks import TaskSet
+from continuum.download import ProgressBar
+
+
+class SegmentationClassIncremental(ClassIncremental):
```

Comment:
C'est pas possible d'utiliser la class ClassIncremental pour la segmentation pour garder ca transparent pour l'utilisateur ?

Reply:
That could be a good idea, but we should be able to do it cleanly. I'm going to think about it

Reply:
As said in private discussion, segmentation brings too many arguments. It'd be more confusing to merge those two classes together.